config/locales/inherited_proofing/en.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Add placeholder es.yml and fr.yml perhaps?

Reply:
Yep, Spanish and French content are required for the build to pass. If this content was copied from the other proofing flows, I assume we can similarly copy that for French and Spanish?